central/role/accessscope_ids.go

```diff
@@ -1,6 +1,8 @@
 package role
 
-// postgres ID for default sccess scopes
+// Postgres IDs for access scopes
+// The values are UUIDs taken in descending order from ffffffff-ffff-fff4-f5ff-ffffffffffff
+// Next ID: ffffffff-ffff-fff4-f5ff-fffffffffffd
 const (
 	denyAllAccessScopeID      = "ffffffff-ffff-fff4-f5ff-fffffffffffe"
```

Comment:
nit:  would it make sense to order these in descending order like was done with permission sets?